federation/pkg/federation-controller/replicaset/replicasetcontroller.go

```diff
@@ -338,9 +358,9 @@ func (frsc *ReplicaSetController) reconcileReplicaSet(key string) error {
 		return nil
 	}
 
-	glog.Infof("Start reconcile replicaset %q", key)
+	glog.V(1).Infof("Start reconcile replicaset %q", key)
```

Comment:
@mwielgus @wojtek-t Whooah!  Why is this at V(1)?  Surely V(4) is more appropriate?
https://github.com/kubernetes/kubernetes/blob/master/docs/devel/logging.md

Same goes for the rest of this PR.